kafkaAdmin/resource_topic_create_test.go

```diff
@@ -26,14 +26,16 @@ func TestAccKafkaAdminTopicCreate(t *testing.T) {
 						"kafka_topic.foo", "partitions", "2"),
 					resource.TestCheckResourceAttr(
 						"kafka_topic.foo", "replication_factor", "3"),
-					/* Not yet implemented
 					resource.TestCheckResourceAttr(
-						"kafka_topic.foo", "cleanup_policy", "1073741824"),
+						"kafka_topic.foo", "cleanup_policy", "compact"),
 					resource.TestCheckResourceAttr(
-						"kafka_topic.foo", "retention_ms", "300000"),
+						"kafka_topic.foo", "retention_ms", "-1"),
 					resource.TestCheckResourceAttr(
 						"kafka_topic.foo", "segment_bytes", "1073741824"),
-					*/
+					resource.TestCheckResourceAttr(
+						"kafka_topic.foo", "segment_ms", "604800000"),
+					resource.TestCheckResourceAttr(
+						"kafka_topic.foo", "retention_bytes", "-1"),
```

Comment:
we accidentally left out  `min_insync_replicas` property, same with createwithconfig test.